<launch>
	<node name="mission_uav_assign" pkg="mission_uav_assign" type="mission_uav_assign" respawn="false" output="screen" clear_params="true">
	<rosparam command="load" file="$(find mission_uav_assign)/launch/start_mission_uav_assign.yaml"/>
	<remap from="/mission_uav_assign/setPoint" to="/mavros/setpoint_position/local" />
	<remap from="/mission_uav_assign/getPose" to="/mavros/local_position/pose" />
	<remap from="/mission_uav_assign/stateUAV" to="/mavros/state" />
	</node>
</launch>
